home/utils.py

```diff
@@ -191,11 +193,20 @@ def add_parent(row, page, home_page):
         else:
             home_page.add_child(instance=page)
 
-    def add_web(row):
+    def add_web(row, page):
```

Comment:
Shouldn't page here have a default, like the `add_whatsapp` definition has `page=None`?